build.go

```diff
@@ -522,3 +529,16 @@ func randString(n int) string {
 	}
 	return string(b)
 }
+
+func buildPlatformVolumes(volumes []string) ([]string, error) {
```

Comment:
I'm wondering if we could skip any translation logic and just pass the strings as-is from the CLI flag to the underlying docker client? Wouldn't any formatting errors be surfaced at that time? If that works, it could simplify the implementation of this feature quite a bit.

Reply:
Oh, I forgot that we need to prepend `platform` to the in-container mount location...